packages/ipfs-core/src/components/dag/get.js

```diff
@@ -14,29 +14,25 @@ module.exports = ({ ipld, preload }) => {
   /**
    * Retrieve an IPLD format node
    *
-   * @param {CID} ipfsPath - A DAG node that follows one of the supported IPLD formats
-   * @param {GetOptions & AbortOptions} [options] - An optional configration
-   * @returns {Promise<DagEntry>}
    * @example
    * ```js
-   * ```JavaScript
    * // example obj
    * const obj = {
-   *   a: 1,
-   *   b: [1, 2, 3],
-   *   c: {
-   *     ca: [5, 6, 7],
-   *     cb: 'foo'
-   *   }
+   * a: 1,
```

Comment:
why did you remove the identation ? 
this might be the linter we should check this out in other files and make sure the examples look good because we want docs to be generated from this stuff

Reply:
Not sure what happened here, but it's fixed now.